apps/ui/src/composables/usePayment.ts

```diff
@@ -52,8 +52,8 @@ export const TOKENS: Record<ChainId, Token[]> = {
   ]
 } as const;
 
-function getWeiAmount(token: Token, amount: number): BigNumber {
-  return BigNumber.from(amount * 10 ** token.decimals);
+function getWeiAmount(token: Token, amount: number): bigint {
+  return BigNumber.from(amount * 10 ** token.decimals).toBigInt();
```

Comment:
To reduce surface area of this refactor just converted `BigNumber` to `bigint` as-is.

In the future I'd consider our own `getWeiAmount` helper that we use everywhere instead of `* 10 ** decimals` and `parseUnits`) throughout the app.

Reply:
Isn't BigInt already capable of computing `10 ** decimals` ?

Reply:
It is, just didn't want to add too many changes in this code unrelated to the refactor (which could be done when we create our own shared helper).